.eslintcache

```diff
@@ -0,0 +1 @@
+{"/Users/wellyngtonamaral/Pessoal/inside-client/commitlint.config.js":{"size":52,"mtime":1507076667000,"hashOfConfig":"1jwvurt","results":{"filePath":"/Users/wellyngtonamaral/Pessoal/inside-client/commitlint.config.js","messages":[],"errorCount":0,"warningCount":0,"fixableErrorCount":0,"fixableWarningCount":0}},"/Users/wellyngtonamaral/Pessoal/inside-client/commitlint/index.js":{"size":780,"mtime":1507076667000,"hashOfConfig":"1jwvurt","results":{"filePath":"/Users/wellyngtonamaral/Pessoal/inside-client/commitlint/index.js","messages":[],"errorCount":0,"warningCount":0,"fixableErrorCount":0,"fixableWarningCount":0}},"/Users/wellyngtonamaral/Pessoal/inside-client/src/components/pages/Home/Home.container.js":{"size":373,"mtime":1507080938000,"hashOfConfig":"1jwvurt","results":{"filePath":"/Users/wellyngtonamaral/Pessoal/inside-client/src/components/pages/Home/Home.container.js","messages":[],"errorCount":0,"warningCount":0,"fixableErrorCount":0,"fixableWarningCount":0}},"/Users/wellyngtonamaral/Pessoal/inside-client/src/components/pages/Home/Home.js":{"size":362,"mtime":1507080950000,"hashOfConfig":"1jwvurt","results":{"filePath":"/Users/wellyngtonamaral/Pessoal/inside-client/src/components/pages/Home/Home.js","messages":[],"errorCount":0,"warningCount":0,"fixableErrorCount":0,"fixableWarningCount":0}},"/Users/wellyngtonamaral/Pessoal/inside-client/src/components/pages/Home/index.js":{"size":59,"mtime":1507080703000,"hashOfConfig":"1jwvurt","results":{"filePath":"/Users/wellyngtonamaral/Pessoal/inside-client/src/components/pages/Home/index.js","messages":[],"errorCount":0,"warningCount":0,"fixableErrorCount":0,"fixableWarningCount":0}},"/Users/wellyngtonamaral/Pessoal/inside-client/src/index.js":{"size":711,"mtime":1507081230000,"hashOfConfig":"1jwvurt","results":{"filePath":"/Users/wellyngtonamaral/Pessoal/inside-client/src/index.js","messages":[],"errorCount":0,"warningCount":0,"fixableErrorCount":0,"fixableWarningCount":0}},"/Users/wellyngtonamaral/Pessoal/inside-client/src/reducers/index.js":{"size":149,"mtime":1507079822000,"hashOfConfig":"1jwvurt","results":{"filePath":"/Users/wellyngtonamaral/Pessoal/inside-client/src/reducers/index.js","messages":[],"errorCount":0,"warningCount":0,"fixableErrorCount":0,"fixableWarningCount":0}},"/Users/wellyngtonamaral/Pessoal/inside-client/src/reducers/test/test.js":{"size":463,"mtime":1507080927000,"hashOfConfig":"1jwvurt","results":{"filePath":"/Users/wellyngtonamaral/Pessoal/inside-client/src/reducers/test/test.js","messages":[],"errorCount":0,"warningCount":0,"fixableErrorCount":0,"fixableWarningCount":0}}}
```

Comment:
I think you should put this guy on the .gitignore

Reply:
done!